change/@fluentui-web-components-be1b97fa-2097-4adc-ad03-f84e0ac0b1b1.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "  \"comment\": feat(button): add button web component",
```

Comment:
```suggestion
  "comment": "feat(button): add button web component",
```

Reply:
Every time - I'll get it one of these times :)

Reply:
I would not be that nit-picky, but this is what populates the changelog.

Reply:
Oh I get it, I just forget lol 😄